src/handlers/issue/generate-erc721-permit-signature.ts

```diff
@@ -89,39 +91,39 @@ export async function generateNftMintSignature({
     throw console.error("Failed to instantiate wallet", error);
   }
 
-  const nftMintRequest: NftMintRequest = {
+  const erc721SignatureData: Erc721PermitSignatureData = {
     beneficiary: beneficiary,
     deadline: MaxUint256,
     keys: keys.map((key) => utils.keccak256(utils.toUtf8Bytes(key))),
-    nonce: BigNumber.from(keccak256(toUtf8Bytes(`${organizationName}${repositoryName}${issueNumber}${username}`))),
+    nonce: BigNumber.from(keccak256(toUtf8Bytes(issueId))),
     values: [organizationName, repositoryName, issueNumber, username, contributionType],
   };
 
-  const signature = await adminWallet._signTypedData(domain, types, nftMintRequest).catch((error) => {
+  const signature = await adminWallet._signTypedData(domain, types, erc721SignatureData).catch((error) => {
     throw console.error("Failed to sign typed data", error);
   });
 
   const nftMetadata: Record<string, string> = {};
 
   keys.forEach((element, index) => {
-    nftMetadata[element] = nftMintRequest.values[index];
+    nftMetadata[element] = erc721SignatureData.values[index];
   });
 
-  const transactionData: NftMintRequestData = {
+  const erc721Data: Erc721PermitTransactionData = {
     request: {
-      beneficiary: nftMintRequest.beneficiary,
-      deadline: nftMintRequest.deadline.toString(),
-      keys: nftMintRequest.keys,
-      nonce: nftMintRequest.nonce.toString(),
-      values: nftMintRequest.values,
+      beneficiary: erc721SignatureData.beneficiary,
+      deadline: erc721SignatureData.deadline.toString(),
+      keys: erc721SignatureData.keys,
+      nonce: erc721SignatureData.nonce.toString(),
+      values: erc721SignatureData.values,
     },
     nftMetadata,
     nftAddress: NFT_CONTRACT_ADDRESS,
     networkId: NFT_CONTRACT_CHAIN_ID,
     signature: signature,
   };
 
-  console.info("Generated nft mint signature", { transactionData });
+  console.info("Generated ERC721 permit signature", { erc721Data });
```

Comment:
Don't you think its more semantically correct to say "Generated ERC721 compliant permit signature"